src/server/req_message.c

```diff
@@ -287,51 +258,21 @@ req_relnodesjob(struct batch_request *preq)
 		return;
 
 	/*
-	 ** Returns job pointer for singleton job or "parent" of
-	 ** an array job.
+	 ** Returns job pointer for singleton job and "parent" of
+	 ** an array sub job.
 	 */
-	pjob = chk_job_request(jid, preq, &jt);
-	if (pjob == NULL) {
+	if (FALSE == chk_job_request(jid, preq, &jt, &js, &pjob, NULL, NULL)) {
 		return;
 	}
 
-	if (jt == IS_ARRAY_NO) {		/* a regular job is okay */
+	if ((jt == IS_ARRAY_NO) || (jt == IS_ARRAY_Single)) {		/* a regular job or a subjob is okay */
 		/* the job must be running */
 		if ((pjob->ji_qs.ji_state != JOB_STATE_RUNNING) ||
 			(pjob->ji_qs.ji_substate !=
 			JOB_SUBSTATE_RUNNING)) {
 			req_reject(PBSE_BADSTATE, 0, preq);
```

Comment:
Same comment as https://github.com/PBSPro/pbspro/pull/590/files/be6187c97c15d46faa1f6ecac4b0512ba08c06d4#r174999607

Reply:
>Earlier to this code change if subjob is in state JOB_STATE_RUNNING and its substate is in JOB_SUBSTATE_PRERUN this combination was allowed. Any reason why this is not allowed now?

Now subjob and normal job will be treated same here.  If you have any use case in mind for not allowing this combination for subjob please point me to it.